cypress/e2e/GUI/gui.cy.js

```diff
@@ -0,0 +1,43 @@
+/// <reference types="Cypress" />
+
+describe('Testando a aplicação EngageSphere', () => {
+
+  
+  beforeEach(() => {
+    // Define uma data e hora fixa
+    const fixedDate = new Date(2024, 6, 10);
+    cy.clock(fixedDate.getTime());
+    cy.visit('http://localhost:3000');
```

Comment:
Visto que a `baseUrl` está definida, aqui ela pode ser omitida.

```suggestion
    cy.visit('/');
```